fusion-cli/package.json

```diff
@@ -101,7 +101,13 @@
   },
   "peerDependencies": {
     "fusion-core": "0.0.0-monorepo",
-    "fusion-tokens": "0.0.0-monorepo"
+    "fusion-tokens": "0.0.0-monorepo",
+    "graphql": "^14.1.1"
+  },
+  "peerDependenciesMeta": {
```

Comment:
This is a feature added in 1.13.0 of yarn and is undocumented as far as I can tell. The effect is that no warning will be emitted when peerDep graphql is not found